flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/ATSU/A320_Neo_CDU_AOC_OFPData.js

```diff
@@ -70,13 +70,21 @@ class CDUAocOfpData {
                     maxValue: station.seats,
                 },
                 async (value) => {
+                    await setDefaultWeights(0, 0);
                     await SimVar.SetSimVarValue(`L:${station.simVar}_DESIRED`, "Number", value);
                     await setTargetCargo(value, '');
                     updateView();
                 }
             );
         }
 
+        async function setDefaultWeights(simbriefPaxWeight, simbriefBagWeight) {
+            const paxWeight = (simbriefPaxWeight === 0) ? 84 : simbriefPaxWeight;
+            const bagWeight = (simbriefBagWeight === 0) ? 20 : simbriefBagWeight;
```

Comment:
Shouldn't these lines be if the corresponding simbrief weights are NaN, i.e., simbrief was not used? If someone wants to set the passenger weight or per passenger baggage weight to zero in order to get some desired payload number, shouldn't we be using that zero value?  Or am I misunderstanding what these lines do?

Reply:
The thing here is we are using the same function to set default weights in both scenarios. When taking values from Simbrief, it is never a NaN, and when manually inputting values, we cannot modify the per pax/bag values till we have further changes on the EFB side. As a fallback we simply set the values to our current default values

Reply:
Ahh, of course.